composer.json

```diff
@@ -8,10 +8,10 @@
     ],
     "license": "MIT",
     "require": {
-        "carsdotcom/laravel-json-schema": "^v1.0.1 || ^v2",
+        "carsdotcom/laravel-json-schema": "^v1.0.1 || ^v2 || dev-bbene-laravel-11",
```

Comment:
Don't merge until I get a new version of laravel-json-schema published and update this,